.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Looks like we need a blanket ignore for `.idea` in the project's base `.gitignore` - these files shouldn't be added.

Reply:
Yeah sorry about that! I'm too used to doing `git add .` and not even checking what's been added.